examples/src/test/scala/scala/meta/serialiser/MappableTest.scala

```diff
@@ -48,6 +49,25 @@ class MappableTest extends WordSpec with Matchers {
     }
   }
 
+  "case class with default" should {
+    "serialise and deserialise" in {
+      val testInstance = WithDefaultValue(s = "something")
+      val keyValue = WithDefaultValue.toMap(testInstance)
+      WithDefaultValue.fromMap(keyValue) shouldBe Some(testInstance)
+    }
+
+    "store correct defaultValueMap" in {
+      val testInstance = WithDefaultValue(s = "something")
+      WithDefaultValue.defaultValueMap shouldBe (Map[String, Any]("i" -> 13))
```

Comment:
oh sorry, i misunderstood. can you delete the line above this (`val testInstance = WithDefaultValue(s = "something")`) to make this more obvious?